charts/osm/values.yaml

```diff
@@ -249,8 +253,12 @@ OpenServiceMesh:
       minReplicas: 1
       # -- Maximum replicas for autoscale
       maxReplicas: 5
-      # -- Average target CPU utilization (%)
-      targetAverageUtilization: 80
+      cpu:
+        # -- Average target CPU utilization (%)
+        targetAverageUtilization: 80
+      memory:
+      # -- Average target value of memory
+        targetAverageValue: "10Mi"
```

Comment:
The memory demands for injector and controller will not be the same, so we should not be using the same value for both. It would make sense to create a unique autoScale property for each of them, nested under the `osmController` and `injector` properties for this reason.

Moreover, I would like suggest using the same units for the memory as specified in the `resource` property for the respective apps. In this case, I suggest defining the memory value as an integer that represents the bytes instead of the power-of-two notation. 
```
    resource:
      limits:
        cpu: "1.5"
        memory: "512M"
      requests:
        cpu: "0.5"
        memory: "128M"
```

Also could you document how the value `10Mi` was chosen?

Reply:
Changed it to be similar to cpu in that if the memory targetAverageUtilization is greater than 80% of the requested memory then it'll scale up